pages/deployment/aws.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Deploy Memgraph on AWS EC2 instance 
+description: Learn how to deploy your Memgraph instance on the AWS EC2 instance.
+---
+
+import { Card, Cards, Tabs, Tab } from 'nextra/components'
+
+# Deploy Memgraph on AWS EC2 instance
+
+This guide will show you how to deploy Memgraph on an AWS EC2 instance. The
+guide will cover only the specific bits that are different from the general
+deployment guide you can find for native [Linux](./linux.mdx) or
+[Docker](./docker.mdx) deployments.
+
+This guide assumes you have an AWS account and are familiar with the AWS
+console. If you are not, you can follow the [AWS EC2
+documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html).
+
+
+## Creating a new EC2 instance 
+
+The first step is to have an EC2 instance running on AWS, where you will deploy
+Memgraph. Here are some guidelines for opening a new EC2 instance for Memgraph. 
+
+### Picking the OS 
+
+When creating the EC2 instance, you need to pick the OS you wish to have on the
+instance. If you are going to use the Memgraph native installation, you need to
+pick a Linux distribution that Memgraph supports.
+
+Memgraph supports multiple Linux distributions. Memgraph packages for supported distributions and versions can be downloaded 
+from the [following page](/getting-started/install-memgraph/direct-download-links#linux)
+
+
+This way, you will manage the native installation of Memgraph, which can bring
+some of the speed benefits. 
+
+If you are going to use the Memgraph Docker image, you can pick any Linux
+distribution you prefer, as long as you can run Docker on it.
+
+
+### Picking the instance type
+
+When creating the EC2 instance, you need to pick the instance type. The process
+highly depends on your actuall workload. For heavy graph processing, consider
+using instances with more CPU and memory. 
+
+If you are running Memgraph in `IN_MEMORY_TRANSACTIONAL` (default mode) or in `IN_MEMORY_ANALYTICAL` storage mode, all data is stored in RAM. That means it would be good to first calculate [how much memory you will
+need](/fundamentals/storage-memory-usage#calculate-storage-memory-usage).
+There is also an [easy-to-use
+calulator](https://memgraph.com/enterprise#ram-calculator) for approximating
+memory usage. The good rule of thumb is to have double the memory of what your
+storage would take. You can go with less if you do not have a demanding
+workload.
+
+
+AWS has specialized instances for [memory-optimized
+workloads](https://docs.aws.amazon.com/ec2/latest/instancetypes/mo.html) which
+are designed for in-memory databases, data analytics, and other memory-intensive
+applications. They also provide DDR5 memory, so you may consider using such instances to
+optimize for better performance. 
+
+
+### Setup the network
```

Comment:
Do we need to talk about VPN access? Or are we assuming that all access will be purely within the cloud?